### PR TITLE
Issue #3273988 by tBKoT: Shows titles corectly on the group invites page

### PR DESCRIPTION
**Pull request by tbkot**
## Problem
When we invited a user to the group to which this user does not have access

## Solution
Create a new class for the `entity reference label` field formatted plugin

## Issue tracker
 - https://www.drupal.org/project/social/issues/3273988
 - https://getopensocial.atlassian.net/browse/YANG-7489

## Theme issue tracker
N/A

## How to test
- [x] Using the latest version of Open Social with the `social_group_invite` module enabled
- [x] As a CM+
- [x] Create a flexible group with the `Invite only` join method
- [x] Go to the Manage members page
- [x] Invite any user to this group
- [x] Log in as an invited user
- [x] Go to the `My invites` -> `Group invites`

### Before merge
- [ ] Code/peer review is completed
- [ ] All commit messages are [clear and clean](https://open-social.slite.com/app/docs/DnmermZDIx_0OQ). If applicable a rebase was performed
- [ ] All automated tests are green
- [ ] Functional/manual tests of the acceptance criteria are approved
- [ ] All acceptance criteria were met
- [ ] If applicable (I.E. new hook_updates) the update path from previous versions (major and minor versions) are tested
- [ ] This pull request has all required labels (team/type/priority)
- [ ] This pull request has a milestone
- [ ] This pull request has an assignee (if applicable)
- [ ] Any front end changes are tested on all major browsers
- [ ] New UI elements, or changes on UI elements are approved by the design team
- [ ] New features, or feature changes are approved by the product owner

### After merge
- [ ] Code is tested on all branches that it has been cherry-picked
- [ ] Update hook number might need adjustment, make sure they have the correct order
- [ ] The Drupal.org ticket(s) are updated according to this pull request status

## Screenshots
зображення

## Release notes
Fixed issue when the user cannot see the group title on the group invites page

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2866-xljpbg3fumj71zfrxyd1xyyo1mbnyx5p.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee6f0d9027707d3862922

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2866-xljpbg3fumj71zfrxyd1xyyo1mbnyx5p.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee6f0d9027707d3862922

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2866-xljpbg3fumj71zfrxyd1xyyo1mbnyx5p.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee6f0d9027707d3862922

**Review by Ressinel — CHANGES_REQUESTED** [review]
Looks good to me.

File: modules/social_features/social_group/modules/social_group_invite/social_group_invite.module [hunk 1]
```diff
@@ -820,3 +821,12 @@ function social_group_invite_preprocess_page(array &$variables): void {
     $variables['page']['complementary_top'] = [];
   }
 }
+
+/**
+ * Implements hook_field_formatter_info_alter().
+ */
+function social_group_invite_field_formatter_info_alter(array &$info): void {
+  if (isset($info['entity_reference_label'])) {
```
[the remark this answers is not in the document]
Comment (dmytriikaiun) [reply modules/social_features/social_group/modules/social_group_invite/social_group_invite.module:835]: Added

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2866-xljpbg3fumj71zfrxyd1xyyo1mbnyx5p.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/624ee6f0d9027707d3862922

**Review by Ressinel — APPROVED** [review]

**Review by rollki — APPROVED** [review]
LGTM.

**Comment by Ressinel** [comment]
🍒-picked to [11.0.x](https://github.com/goalgorilla/open_social/commit/4f57d200ac8b6b07aaa31543248a493e8088c220), [11.1.x](https://github.com/goalgorilla/open_social/commit/6392c5f77e9bf9a2def3b7b6fce562e089e94faf), and [11.2.x](https://github.com/goalgorilla/open_social/commit/1c63ccaead257c29cd5d8d6f4a9958d6d4d4c9b2)

**Pull request merged**